src/common/options/crimson.yaml.in

```diff
@@ -2,53 +2,82 @@
 ---
 
 options:
-- name: crimson_osd_obc_lru_size
-  type: uint
-  level: advanced
-  desc: Number of obcs to cache
-  default: 512
-- name: crimson_osd_scheduler_concurrency
+
+# seastar CPU pinning (only 1 option applies):
+
+- name: crimson_seastar_num_threads
   type: uint
   level: advanced
-  desc: The maximum number concurrent IO operations, 0 for unlimited
   default: 0
-- name: crimson_alien_op_num_threads
-  type: uint
-  level: advanced
-  desc: The number of threads for serving alienized ObjectStore
-  default: 6
+  desc: The number of threads for serving seastar reactors without CPU pinning, overridden if crimson_seastar_cpu_cores is set.
+  long_desc: The number of threads for serving seastar reactors without CPU pinning,
+             overridden if crimson_seastar_cpu_cores is set.
+             smp::count is deduced from this option.
   flags:
   - startup
+  min: 0
+  max: 32
```

Comment:
How about 256?  32 is pretty low.

Reply:
Did you mean to comment on `crimson_alien_op_num_threads` or `crimson_seastar_num_threads`? Since it's hard to tell from the diff.
For `crimson_alien_op_num_threads` there's no limit (min/max).
For `crimson_seastar_num_threads` - afaiu, we shouldn't expect for more than 32 reactors per OSD atm, right?

I'm moving this commit into https://github.com/ceph/ceph/pull/65007.